source/guides/administrator.rst

```diff
@@ -136,7 +136,8 @@ Learn how to maintain your Mattermost system.
    /administration/health-check*
    /administration/announcement-banner.rst
    /administration/bulk-export.rst
- 
+   /adminstration/product-recipes-information-architecture.rst
```

Comment:
See comment here: https://github.com/mattermost/docs/pull/2741#issuecomment-497034900.

These docs are actually internal processes, so a more suitable place for these would be in https://docs.mattermost.com/process/documentation-guidelines.html.